io.kapsa.drive.json

```diff
@@ -0,0 +1,365 @@
+      ],
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/man"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libsecret/0.20/libsecret-0.20.5.tar.xz",
+          "sha256": "3fb3ce340fcd7db54d87c893e69bfc2b1f6e4d4b279065ffe66dac9f0fd12b4d"
+        }
+      ]
+    },
+    {
+      "name": "libsodium",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/jedisct1/libsodium/archive/1.0.18-RELEASE.tar.gz",
+          "sha256": "b7292dd1da67a049c8e78415cd498ec138d194cfdb302e716b08d26b80fecc10",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1728,
+            "url-template": "https://github.com/jedisct1/libsodium/archive/$version.tar.gz"
+          }
+        }
+      ],
+      "post-install": [
+        "install -Dm644 LICENSE ${FLATPAK_DEST}/share/licenses/libsodium/LICENSE"
+      ]
+    },
+    {
+      "name": "s3drive",
+      "only-arches": [
+        "x86_64"
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "install apply_extra ${FLATPAK_DEST}/bin",
+        "install s3drive.sh ${FLATPAK_DEST}/bin/s3drive",
+        "install -Dm644 io.kapsa.drive.metainfo.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.metainfo.xml",
+        "install -Dm644 s3drive_logo.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png",
+        "install -Dm644 io.kapsa.drive.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
+        "install -Dm755 /usr/bin/ar -t ${FLATPAK_DEST}/bin",
+        "install -Dm755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t ${FLATPAK_DEST}/lib"
+      ],
+      "sources": [
+        {
+          "type": "script",
+          "dest-filename": "apply_extra",
+          "commands": [
+            "ar x s3drive_amd64.deb",
+            "rm -f s3drive_amd64.deb",
+            "tar --use-compress-program=unzstd -xvf data.tar.zst",
+            "rm -f control.tar.zst data.tar.zst debian-binary",
+            "mv usr/* .",
+            "chmod -R a-s,go+rX,go-w .",
+            "rmdir usr"
+          ]
+        },
+        {
+          "type": "script",
+          "dest-filename": "s3drive.sh",
+          "commands": [
+            "exec env TMPDIR=$XDG_CACHE_HOME /app/extra/local/lib/s3drive/kapsa -s $EXTRA_ARGS \"$@\""
+          ]
+        },
+        {
+          "type": "file",
+          "path": "io.kapsa.drive.metainfo.xml"
+        },
+        {
+          "type": "file",
+          "path": "io.kapsa.drive.desktop"
+        },
+        {
+          "type": "file",
+          "path": "s3drive_logo.png"
+        },
+        {
+          "type": "extra-data",
+          "filename": "s3drive_amd64.deb",
+          "only-arches": [
+            "x86_64"
+          ],
+          "url": "https://github.com/s3drive/app/releases/latest/download/s3drive_amd64.deb",
```

Comment:
this unversioned URL for downloading the software will cause a lot of grief.

Reply:
Sorry, I think I don't exactly understand it all exactly. The latest versioned URL is: https://github.com/s3drive/app/releases/download/1.5.1/s3drive_amd64.deb

Does it mean I will have to updated it each new release?

Reply:
Changed to:  https://github.com/s3drive/app/releases/download/1.5.1/s3drive_amd64.deb